src/librustdoc/clean/auto_trait.rs

```diff
@@ -34,7 +34,7 @@ impl<'a, 'tcx> AutoTraitFinder<'a, 'tcx> {
         ty: Ty<'tcx>,
         trait_def_id: DefId,
         param_env: ty::ParamEnv<'tcx>,
```

Comment:
Why does this still take a `param_env` given that the other places call `tcx.param_env` on the `item_def_id`?

Reply:
This was added after I opened the original PR, I'll fix it.

Reply:
Hmm, actually this is called in a loop, I don't know if it makes sense to change. It will have to lookup the type/param_env each time, even though it's memoized that still has overhead.